cms/static/sass/edx-icons.scss

```diff
@@ -0,0 +1,15 @@
+// studio - edx-icons
+// ====================
+
+// Table of Contents
+// * +Libs and Resets
+// * +Vendor and Rebase
+// * +edx-fonts
+
+
+// +Libs and Resets - *do not edit*
```

Comment:
Scary! Can you add a comment explaining what's going on here and why one should avoid editing it?